src/Microsoft.Health.SqlServer/Microsoft.Health.SqlServer.csproj

```diff
@@ -14,9 +14,11 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.2.3" />
```

Comment:
Where is this reference used?

Reply:
Thanks for catching this. I added this earlier while trying something and forgot to remove it.

Reply:
Done.